kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/internal/MergeImpl.kt

```diff
@@ -17,46 +17,55 @@ internal abstract class KoverMergingConfigImpl: KoverMergingConfig {
     internal var instrumentationAction: Action<KoverMergingInstrumentation>? = null
     internal val variantsAction: MutableMap<String, Action<KoverMergingVariantCreate>> = mutableMapOf()
 
+    internal var configured: Boolean = false
```

Comment:
I think some test is needed for this new boolean property, with e.g., `kover.merge.subprojects()` in configuration

Reply:
It's already tested, see tests in `MergingTests`, e.g. `testRootNoProjects`

Reply:
I mean you need to test that syntax `kover.merge.subprojects` works the same as `kover { merge { subprojects() } }`